electionleaflets/settings/base.py

```diff
@@ -1,32 +1,29 @@
-import sys
-from os.path import join, abspath, dirname
+import environ, sys
```

Comment:
Small general python point: this is not PEP8 compatible, imports should be one per line